### ISSUE TITLE
Jump to next bookmark gets stuck

### ISSUE THREAD

**Issue by pdonias** [atom/bookmarks#75]
Jumping to the next bookmark doesn't work when the bookmarks have been added in a certain order.
Let me clarify that:

When I add the bookmarks in this order (line 1 --> line 3), everything works fine:
![atombug_keys](https://cloud.githubusercontent.com/assets/10992860/20558833/57217dbc-b172-11e6-863e-3da241baf0bd.gif)

But when I add them in this order (line 3 --> line 1), it gets stuck to line 3:
![atombug_keys2](https://cloud.githubusercontent.com/assets/10992860/20558857/7fd646f2-b172-11e6-897b-7504f160536f.gif)

After further testing, it seems that it gets stuck on the **first added bookmark** if it's the **last bookmark in the file**.

**Comment by Ben3eeE** [comment]
Thanks for taking the time to contribute!

We noticed that this is a duplicate of https://github.com/atom/bookmarks/issues/69, which has been fixed in Atom 1.13.0. ✨ 

Because we treat our issues list as [the Atom team's backlog](https://en.wikipedia.org/wiki/Scrum_(software_development)#Product_backlog), we close duplicates to focus our work and not have to touch the same chunk of code for the same reason multiple times. This is also why we may mark something as duplicate that isn't an exact duplicate but is closely related.

For information on how to use GitHub's search feature to find out if something is a duplicate before filing, see [the How Can I Contribute? section of the Atom CONTRIBUTING guide](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#how-can-i-contribute).
